bin/pfatt.sh

```diff
@@ -228,4 +229,4 @@ elif [ "$EAP_MODE" = "supplicant" ] ; then
 else
   /usr/bin/logger -st "pfatt" "error: unknown EAP_MODE. '$EAP_MODE' is not valid. exiting..."
   exit 1
-fi
\ No newline at end of file
+fi
```

Comment:
Sorry, I'm using the GitHub web editor. I can adjust this on the command line to remove the unrelated whitespace change.